app/models/micropost.rb

```diff
@@ -0,0 +1,15 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope ->{order(created_at: :desc)}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
+  validates :content, presence: true, length:
+    {maximum: Settings.micropost.max_content}
+  validate  :picture_size
+
+  private
+  def picture_size
+    return errors.add(:picture, t(".img_less_than")) if
```

Comment:
nếu dùng if mà dài quá so với 1 hàng thì dùng if..end luôn đi e

Reply:
anh ơi, nếu dùng if... end thì robocop bắt lỗi này anh. Use a guard clause instead of wrapping the code inside a conditional expression.

Reply:
def picture_size
    return unless picture.size > Settings.micropost.img_size.megabytes
    errors.add(:picture, t(".img_less_than"))
  end
em viết lại như thế này được không anh ?